src/prost/build.rs

```diff
@@ -112,6 +112,57 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
             "#[derive(::enum_as_inner::EnumAsInner)]",
         )
         .btree_map(btree_map_paths)
+        // node body is a very large enum, so we box it to avoid stack overflow.
+        // TODO: ideally we should box all enum variants automatically https://github.com/tokio-rs/prost/issues/1209
+        .boxed(".stream_plan.StreamNode.node_body.source")
```

Comment:
Why not `.boxed(".stream_plan.StreamNode")`?

Reply:
Sounds can also fix the stackoverflow. 🤔 But the enum itself still wastes memory.

Reply:
Listing all possible variants here seems verbose. Is this the only approach to achieve the goal?

Reply:
can implement this upstream https://github.com/tokio-rs/prost/issues/1209